salt/_modules/metalk8s_drain.py

```diff
@@ -228,8 +228,7 @@ def get_controller(self, namespace, controller_ref):
             kind=controller_ref['kind'],
             apiVersion=controller_ref['api_version'],
             namespace=namespace,
-            kubeconfig=self._kwargs.get('kubeconfig'),
-            context=self._kwargs.get('context')
+            **self._kwargs
```

Comment:
I thought you didn't want to do this because `self._kwargs` are the `Drain` object's kwargs? Minor though, feel free to leave them.

Reply:
But since we accept kwargs everywhere I think it's ok, at the beginning in kubernetes module we didn't have any kwargs I added it at the end (after your first comment so btw I should have done this before)